test/_external/test262/skiplist

```diff
@@ -0,0 +1,125 @@
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-rqstd-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-self-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-export-dflt-expr-err-eval.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-gtbndng-indirect-update-dflt.js
+
+# export * from ns
+test/language/module-code/eval-rqstd-once.js
+
+# export * from ns
+test/language/module-code/eval-rqstd-order.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-self-abrupt.js
+
+# export * from ns
+test/language/module-code/eval-self-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-bndng-fun.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-bndng-gen.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-bndng-var.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-iee-cycle.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-star-cycle.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-iee-trlng-comma.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-bndng-dflt-named.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-bndng-fun.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-bndng-gen.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-bndng-trlng-comma.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-bndng-var.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-named-iee-cycle.js
+
+# export * from ns
+test/language/module-code/instn-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/instn-star-ambiguous.js
+
```

Comment:
👆 This is a bug (patching)

Reply:
Patched https://github.com/standard-things/esm/commit/599b0df265c38445051a4d4923cad3157d70824a